src/status_im/ui/screens/activity_center/notification/contact_request/view.cljs

```diff
@@ -0,0 +1,58 @@
+(ns status-im.ui.screens.activity-center.notification.contact-request.view
```

Comment:
FYI: Don't worry about reviewing this whole file. It has been extracted from another file after a restructure.